microsoft/PowerToys
# [PowerToys Run] Obsolete APIs and warnings introduced in .net5

## Summary of the Pull Request

**What is this about:**

**What is include in the PR:** 
- Use `WoxJsonStorage` instead of `BinaryStorage`
- Remove `BinaryStorage` and `BinaryFormatter` as a result
- Removed `Window.IsOwner` property as it always returns true. That option doesn't make sense because we want to query all windows not only windows created by `PowerLauncher.exe` process
- Don't include `ImageUsageCache.json` into the bug report tool as it is not needed and can be considered as sensitive info


**How does someone test / validate:** 
- Test PT Run for regressions
- Run unit tests
- Check that `ImageUsageCache.json` is created and populated with valid data

## Quality Checklist

- [X] **Linked issue:** #12285

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
